styles.css

```diff
@@ -137,3 +137,545 @@ a:hover {
   float: left;
   gap: 10px;
 }
+
+section.portfolio-1 {
+  min-height: 718px;
+  padding-top: 100px;
+}
+
+.project-1 {
+  display: flex;
+  flex-direction: column;
+  padding-left: 10px;
+  margin: 12px 0;
+}
+
+.project-name-1 {
+  display: flex;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  padding-bottom: 30px;
+}
+
+.about-project-1 ul {
+  list-style: none;
+  float: left;
+  display: flex;
+  gap: 30px;
+  justify-content: space-around;
+}
+
+.company {
+  color: black;
+  font-weight: bold;
+}
+
+.position {
+  list-style-type: disc;
+  color: gray;
+  font-weight: bold;
+}
+
+.year {
+  list-style-type: disc;
+  color: gray;
+  font-weight: bold;
+}
+
+.role-project-1 {
+  display: flex;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  margin: 12px 0;
+}
+
+.skills-project-1 ul {
+  display: flex;
+  list-style: none;
+  float: left;
+  gap: 30px;
+  flex-direction: row;
+}
+
+.skills-project-1 li {
+  display: flex;
+  align-items: center;
+  padding: 4px 12px;
+  width: 53px;
+  height: 24px;
+  left: 0;
+  top: 0;
+  background-color: #ebebff;
+  border-radius: 8px;
+  color: #6070ff;
+}
+
+.view-project-1 button {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  background-color: white;
+  width: 130px;
+  height: 50px;
+  border-color: #396df2;
+  border-radius: 8px;
+}
+
+section.portfolio-2 {
+  height: 718px;
+}
+
+.project-2 {
+  display: flex;
+  flex-direction: column;
+  padding-left: 10px;
+  margin: 12px 0;
+}
+
+.project-name-2 {
+  display: flex;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  padding-bottom: 30px;
+}
```

Comment:
- [ ] Let's take this card as a reference to all remaining, please, to stick to the Figma design, you need to match the Figma properties especially (`margin, padding, font-size, color, font-weight, font-family) and so on.
see these images below ⬇️  and correct them to match the original design.
## introduction section
- consider adjusting your font-size and color

- [ ] You

<img width="319" alt="yh1" src="https://user-images.githubusercontent.com/79058364/153621260-01787d68-3b39-48b8-ac57-b93ab2bd5de9.png">

<img width="326" alt="ututu" src="https://user-images.githubusercontent.com/79058364/153621820-08a182c3-77cd-450f-852a-913a7d2cef78.png">


- [ ] Figma
<img width="258" alt="fgh1" src="https://user-images.githubusercontent.com/79058364/153621510-e1a33653-24d7-4a29-bc4b-6ccf010ebbb1.png">

<img width="260" alt="letfig" src="https://user-images.githubusercontent.com/79058364/153621693-800dcd42-d862-45aa-8553-1a27a4fdec7a.png">



- [ ] YOUR DESING

<img width="317" alt="car yu" src="https://user-images.githubusercontent.com/79058364/153620466-882e3862-673c-429e-9c50-32fe0fa2fa3f.png">

- [ ] THE FIGMA DESIGN

<img width="223" alt="car fig" src="https://user-images.githubusercontent.com/79058364/153620634-48d77158-365e-409e-8267-4f00066f60e1.png">

- As you can see, please adjust them to match the Figma, remember to check the disposition of buttons too 😺